addons/sale_loyalty_delivery/tests/test_free_shipping_reward.py

```diff
@@ -315,9 +315,9 @@ def test_free_shipping_reward_last_line(self):
         claimable_rewards = order._get_claimable_rewards()
         self.assertEqual(len(claimable_rewards), 1)
         # Try to apply the loyalty card to the sale order
-        self._apply_promo_code(order, loyalty_card.code)
+        self._claim_reward(order, loyalty_program, loyalty_card.code)
```

Comment:
Why do you need to change this? Also, don't you need to enclose it with an `assertTrue`?

Reply:
The apply promo code function is used when user puts the code themselves to redeem the reward, which in case of my changes is no longer desired functionality. The reward is still shown and redeemable, but not from that point. The assertTrue part seems reasonable.

Reply:
I checked, if loyalty_card was not claimable it would return an error, just like _apply_promo_code would, so it seems there is no need to put assertTrue.

Reply:
Ok for the change of method, I think it makes sense.
I'd still add the `assertTrue`, because `_claim_reward` may return `False` in some conditions. In this case, no error, the test is green, but it doesn't test what needs to be tested.